src/core/services/gesture/gesture.js

```diff
@@ -318,6 +331,19 @@ function MdGesture($$MdGestureHandler, $$rAF, $timeout) {
       }
     });
 
+  function getTouchAction() {
+    var testEl = angular.element('<div>')[0];
+    var vendorPrefixes = ['', 'webkit', 'Moz', 'MS', 'ms', 'o'];
```

Comment:
I know what you mean, this prefix looks not valid. But this logic has been ported from HammerJS and it seems to have a role. Otherwise we should file an issue and ask there.